src/js/actions/type.js

```diff
@@ -264,7 +265,7 @@ define(function (require, exports) {
             playObject = [playObject].concat(setOpacityPlayObjects);
         }
         var updatePromise = this.transfer(updateColor, document, layers, color, coalesce, true),
-            setColorPromise = locking.playWithLockOverride(document, layers, playObject, typeOptions);
+            setColorPromise = layerActionsUtil.playSimpleLayerActions(document, layers, playObject, true, typeOptions);
```

Comment:
Given that you've fixed applyColor in libraries, and you only pass text layers to this function, this change should be unnecessary.